### DISCUSSION TITLE
Dark-mode

### DISCUSSION THREAD

**Discussion by Jci-Dev** [primefaces/community#2391] [PrimeVue]
Team, 
I'm a newbie using your controls and vue as well. How can I after doing?
const themeMode = localStorage.getItem("kt_theme_mode_value") === "dark" ? "system" : "light";

app.use(PrimeVue, {
    inputVariant: 'filled',
    theme: {
        preset: Aura,
        options: {
            prefix: 'p',
            darkModeSelector: themeMode,
            cssLayer: false
        }
    }
});

switching mode on my app also make primevue controls change to dark mode?

Thanks in advance,

Vic

**Comment by j0rgedev** [comment]
Hello @Jci-Dev. Please follow this example to switch between light and dark mode used in [Sakai Template](https://github.com/primefaces/sakai-vue/blob/master/src/layout/composables/layout.js#L52).
